test/python/opflow/test_gradients.py

```diff
@@ -288,8 +296,12 @@ def test_state_gradient3(self, method):
         a = Parameter("a")
         # b = Parameter('b')
         params = a
-        x = Symbol("x")
-        expr = cos(x) + 1
+        if HAS_SYMENGINE:
```

Comment:
looks like this can be removed as well since `cos` is a method of ParameterExpression e.g. `numpy.cos(x)+1` should work

Reply:
Good catch, updated in: https://github.com/Qiskit/qiskit-terra/pull/6270/commits/3b4c5ec1d1d81845d5aa3a7f984499946ad8ff13